src/core_plugins/console/public/src/es.js

```diff
@@ -7,6 +7,25 @@ export function getVersion() {
   return esVersion;
 }
 
+export function getContentType(body) {
+  if (!body) return;
+
+  let contentType;
+  try {
+    // there is more than one line
+    const lines = body.split('\n').filter(Boolean);
+    if (!lines.length) throw new Error('not multiline json')
```

Comment:
Calling `split` with a substring that doesn't have any matches results in an array of length 1, so this multiline error check will never fire.  As a result, even a simple:
```
POST logstash-*/_search
{}
```
results in an error